universal-login-wallet/src/services/TransferService.ts

```diff
@@ -22,7 +23,7 @@ class TransferService {
         return this.transferTokens(transferDetails);
       }
     }
-    throw new Error('No user wallet provided!');
+    throw new UserWalletNotFound();
```

Comment:
You can use `ensure` instead: `ensure(this.walletService.userWallet, UserWalletNotFound)`

Reply:
But in such case I would need to cast it explicitly to boolean.

Reply:
ensureNotNull :) nevermind